root/views/layout.sml

```diff
@@ -7,17 +7,17 @@ html
       meta(http-equiv='X-UA-Compatible' content='IE=edge, chrome=1')
       meta(name='description' content='')
       meta(name='author' content='<%= github_username %>')
-      //- uncomment if your site is responsive!
-      //- meta(name="viewport" content="width=device-width, initial-scale=1")
+      meta(name="viewport" content="width=device-width, initial-scale=1")
 
     block(name='title')
       title Title of Your Site
 
     block(name='stylesheets')
       link(rel='stylesheet' href='css/index.css')
 
-  body
-    block(name='content')
+  body(role='document')
```

Comment:
It's going to be `document`, since it's a static site generator.  Even if there are interactive elements, the user can presumably still interact with the site with javascript disabled.

Of course, users may opt to replace the `role` with `application`, if they decide to create a single-page app, but I would think that kind of case is few and far in between the norm.